public/index.css

```diff
@@ -504,6 +504,8 @@ h2.heading {
 /* Highlight TOC header link matching the current scroll position */
 .current-header-link {
 	background-color: var(--theme-bg-accent);
+	/* Indicates the current heading for forced colors users, transparent is changed to a solid color */
+	outline: 1px solid transparent;
```

Comment:
I used invisible `outline`s mainly because they have more support than the `forced-colors` media query and also the fact that they don't increase the element's width/height. Maybe we can go the progressive enhancement route and change the outline for a border for users with browsers that support `forced-colors`, what do you think?

Thanks for the review!